.husky/pre-commit

```diff
@@ -0,0 +1,9 @@
+# .husky/pre-commit
```

Comment:
I'm not comfortable with this pre-commit hook. I often commit things in a broken state, just to have a checkpoint, and a hook like this will either prevent me from doing that or will force me to look up git flags to ignore the precommit hooks.

could we rely just on CI to run these tests? or run them on pre-push like prettier?

Reply:
Done. I am also not happy about the fact that it doesn't run when ran from VS Code. It is a known issue:
https://stackoverflow.com/questions/71423089/vs-code-issue-with-husky